src/App/Pages/Accounts/HomePageViewModel.cs

```diff
@@ -140,5 +152,17 @@ public async Task ContinueToLoginStepAsync()
                 await _platformUtilsService.ShowDialogAsync(AppResources.GenericErrorMessage, AppResources.AnErrorHasOccurred, AppResources.Ok);
             }
         }
+
+        private async Task PromptToSwitchToExistingAccountAsync(string userId)
+        {
+            var switchToAccount = await _platformUtilsService.ShowDialogAsync(
+                AppResources.SwitchToAlreadyAddedAccountConfirmation,
+                AppResources.AccountAlreadyAdded, AppResources.Yes, AppResources.Cancel);
+            if (switchToAccount)
+            {
+                await _stateService.SetActiveUserAsync(userId);
+                _messagingService.Send("switchedAccount");
+            }
+        }
```

Comment:
Please don't duplicate code, this is the same as in `LoginPageViewModel`. Move this to `AccountsManager` so that it can be reused in both places